code/game/gamemodes/steal_items.dm

```diff
@@ -132,6 +132,11 @@ datum/theft_objective/ai/check_special_completion(var/obj/item/device/aicard/C)
 	typepath = /obj/item/clothing/gloves/color/black/krav_maga/sec
 	protected_jobs = list("Head Of Security", "Warden")
 
+/datum/theft_objective/forensics_gloves
```

Comment:
Doesn't really seem like either of those to be honest. Just rare as they are the only ones on the station. I think they should be fine for theft, how many things do you have on that table anyways?